### ISSUE TITLE
@see annotation not supported

### ISSUE THREAD

**Issue by POPSuL** [ApiGen/ApiGen#864]
On master branch:

```
[popsul] ~/projects/ApiGen [master]  % ./bin/apigen generate -vvv --destination=/var/www/api /var/www/core    

                                                                           
  [InvalidArgumentException]                                               
  "\https://github.com/mikey179/vfsStream/issues/6" is not a valid Fqsen.  
                                                                           

Exception trace:
 () at /home/popsul/projects/ApiGen/vendor/phpdocumentor/reflection-common/src/Fqsen.php:44
 phpDocumentor\Reflection\Fqsen->__construct() at /home/popsul/projects/ApiGen/vendor/phpdocumentor/type-resolver/src/FqsenResolver.php:69
 phpDocumentor\Reflection\FqsenResolver->resolvePartialStructuralElementName() at /home/popsul/projects/ApiGen/vendor/phpdocumentor/type-resolver/src/FqsenResolver.php:32
 phpDocumentor\Reflection\FqsenResolver->resolve() at /home/popsul/projects/ApiGen/vendor/phpdocumentor/reflection-docblock/src/DocBlock/Tags/See.php:59
 phpDocumentor\Reflection\DocBlock\Tags\See::create() at n/a:n/a
 call_user_func_array() at /home/popsul/projects/ApiGen/vendor/phpdocumentor/reflection-docblock/src/DocBlock/StandardTagFactory.php:196
 phpDocumentor\Reflection\DocBlock\StandardTagFactory->createTag() at /home/popsul/projects/ApiGen/vendor/phpdocumentor/reflection-docblock/src/DocBlock/StandardTagFactory.php:116
 phpDocumentor\Reflection\DocBlock\StandardTagFactory->create() at /home/popsul/projects/ApiGen/vendor/phpdocumentor/reflection-docblock/src/DocBlockFactory.php:231
 phpDocumentor\Reflection\DocBlockFactory->parseTagBlock() at /home/popsul/projects/ApiGen/vendor/phpdocumentor/reflection-docblock/src/DocBlockFactory.php:96
 phpDocumentor\Reflection\DocBlockFactory->create() at /home/popsul/projects/ApiGen/packages/Reflection/src/Transformer/BetterReflection/Class_/ClassReflectionTransformer.php:53
 ApiGen\Reflection\Transformer\BetterReflection\Class_\ClassReflectionTransformer->transform() at /home/popsul/projects/ApiGen/packages/Reflection/src/TransformerCollector.php:70
 ApiGen\Reflection\TransformerCollector->transformSingle() at /home/popsul/projects/ApiGen/packages/Reflection/src/TransformerCollector.php:43
 ApiGen\Reflection\TransformerCollector->transformGroup() at /home/popsul/projects/ApiGen/packages/Reflection/src/Parser/Parser.php:104
 ApiGen\Reflection\Parser\Parser->transformBetterClassInterfaceAndTraitReflections() at /home/popsul/projects/ApiGen/packages/Reflection/src/Parser/Parser.php:110
 ApiGen\Reflection\Parser\Parser->parseClassElements() at /home/popsul/projects/ApiGen/packages/Reflection/src/Parser/Parser.php:55
 ApiGen\Reflection\Parser\Parser->parseDirectories() at /home/popsul/projects/ApiGen/src/Application/ApiGenApplication.php:55
 ApiGen\Application\ApiGenApplication->runCommand() at /home/popsul/projects/ApiGen/src/Console/Command/GenerateCommand.php:50
 ApiGen\Console\Command\GenerateCommand->execute() at /home/popsul/projects/ApiGen/vendor/symfony/console/Command/Command.php:264
 Symfony\Component\Console\Command\Command->run() at /home/popsul/projects/ApiGen/vendor/symfony/console/Application.php:869
 Symfony\Component\Console\Application->doRunCommand() at /home/popsul/projects/ApiGen/vendor/symfony/console/Application.php:223
 Symfony\Component\Console\Application->doRun() at /home/popsul/projects/ApiGen/vendor/symfony/console/Application.php:130
 Symfony\Component\Console\Application->run() at /home/popsul/projects/ApiGen/bin/apigen:17

generate [--destination DESTINATION] [--] <source> (<source>)...
```

```
[popsul] ~/projects/ApiGen [master]  % grep -HR 'm/issues/6' /var/www/core                                   
/var/www/core/vendor/mikey179/vfsStream/src/test/php/org/bovigo/vfs/vfsStreamWrapperFlockTestCase.php: * @see         https://github.com/mikey179/vfsStream/issues/6
/var/www/core/vendor/mikey179/vfsStream/src/main/php/org/bovigo/vfs/vfsStreamFile.php:     * @see     https://github.com/mikey179/vfsStream/issues/6
/var/www/core/vendor/mikey179/vfsStream/src/main/php/org/bovigo/vfs/vfsStreamFile.php:     * @see     https://github.com/mikey179/vfsStream/issues/6
/var/www/core/vendor/mikey179/vfsStream/src/main/php/org/bovigo/vfs/vfsStreamFile.php:     * @see     https://github.com/mikey179/vfsStream/issues/6
/var/www/core/vendor/mikey179/vfsStream/src/main/php/org/bovigo/vfs/vfsStreamFile.php:     * @see     https://github.com/mikey179/vfsStream/issues/6
/var/www/core/vendor/mikey179/vfsStream/src/main/php/org/bovigo/vfs/vfsStreamWrapper.php:     * @see     https://github.com/mikey179/vfsStream/issues/6
/var/www/core/vendor/nikic/php-parser/test/PhpParser/ParserTest.php:            // (https://github.com/facebook/hhvm/issues/6970)
```

**Comment by TomasVotruba** [comment]
Since ApiGen 5.0 we delegace this to [phpDocumentor/TypeResolver](https://github.com/phpDocumentor/TypeResolver). It uses simple and clear way to use these 2 annotations:

`@see` - for elemetns
`@link` - for urls

See [part in documentation](https://github.com/phpDocumentor/TypeResolver#on-types-and-element-) for more

**Comment by POPSuL** [comment]
@TomasVotruba yes, you're right! `@see` for elements is more correct than `@link`.
But, I can't generate **all** documentation due to an error in third-party library.
Maybe invalid annotations should be skipped (with warning output to console)?
Current behavior like this: "`@return invalidType` is invalid, but we don't skip that and you get exception instead _partial_ documentation".

**Comment by TomasVotruba** [comment]
@POPSuL I understand that. The thing is, it is nested in BetterReflection, that uses TypeResolver. So I'd try to solve that there - **at least a create and issue and report this**. 

Rather than create a new abstract level where to catch these exceptions. If it fails, we can try to make some abstraction level here to allow these failures.

**Comment by TomasVotruba** [comment]
@POPSuL I wonder. How did you solve this? Did you report issue or changed annotations?

**Comment by POPSuL** [comment]
@TomasVotruba I did not solve this problem in any way

**Comment by TomasVotruba** [comment]
Closed by #952
